test/Microsoft.DotNet.Tools.Tests.Utilities/project.json

```diff
@@ -4,21 +4,21 @@
     "compilationOptions": {
         "keyFile": "../../tools/Key.snk"
     },
-    "dependencies": {
-        "NETStandard.Library": "1.0.0-rc2-23811",
-        "System.Collections.Immutable": "1.2.0-rc2-23811",
-        "FluentAssertions": "4.0.0",
-        "xunit": "2.1.0",
-        "dotnet-test-xunit": "1.0.0-dev-48273-16",
+  "dependencies": {
```

Comment:
Is there a standard on .json file formatting?  I seem to fight a lot with this since my settings are set to 4 spaces.  My opinion is that we should use the default VS settings for "General Development".